chall05/sadawi.rb

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env ruby
+#ruby 2.5.1p57
+
+require 'open-uri'
+begin
+file = open('https://en.wikipedia.org/wiki/' + ARGV[0])
```

Comment:
be careful of the indentation even if it;'s not important in Ruby, it's better to keep using it